clodius/models/tileset_info.py

```diff
@@ -0,0 +1,10 @@
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+
+class TilesetInfo(BaseModel):
```

Comment:
I see this is only used in the test. Should that be in the test module only or is this intended as a part of the public API?